<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPasswordDialog</class>
 <widget class="QDialog" name="KPasswordDialog">
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="vBoxLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hboxLayout">
     <item>
      <widget class="QLabel" name="pixmapLabel">
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="prompt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Supply a username and password below.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Ignored</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>400</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="KTitleWidget" name="errorMessage"/>
   </item>
   <item>
    <widget class="QRadioButton" name="anonymousRadioButton">
     <property name="text">
      <string comment="@option:radio">No password, use anonymous (or &amp;guest) login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="usePasswordButton">
     <property name="text">
      <string comment="@option:radio">Use this password:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="credentialsGroup" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string comment="@label:textbox">Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="userEdit">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="userEditContextHelpButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="domainLabel">
        <property name="text">
         <string comment="@label:textbox">Domain:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="domainEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string comment="@label:textbox">Password:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KPasswordLineEdit" name="passEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="keepCheckBox">
        <property name="text">
         <string comment="@option:check">Remember password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>anonymousRadioButton</tabstop>
  <tabstop>usePasswordButton</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>domainEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>keepCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>192</y>
    </hint>
  </hints>
  </connection>
  <connection>
   <sender>usePasswordButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>credentialsGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usePasswordButton</sender>
   <signal>clicked()</signal>
   <receiver>credentialsGroup</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
